src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculetion {

```

Comment:
The interface name `AreaCalculetion` is a verb, which is not a good practice for naming interfaces. Consider renaming it to something like `AreaCalculator` to make it clear that it's an entity that performs a calculation.